desktop/package.json

```diff
@@ -86,5 +88,8 @@
     "webpack-dev-server": "1.14.1",
     "webpack-hot-middleware": "2.10.0",
     "webpack-target-electron-renderer": "0.4.0"
+  },
+  "optionalDependencies": {
+    "fsevents": "1.0.11"
```

Comment:
i think this is related to the known z_buff issue which is an intermittent (and kinda common) thing that happens on linux (sometimes other platforms). if you keep doing npm i it'll work